pkg/cmd/infra/router/template.go

```diff
@@ -287,6 +293,77 @@ func parseCaptureHeaders(in string) ([]templateplugin.CaptureHTTPHeader, error)
 	return captureHeaders, nil
 }
 
+func sanitizeHeaderValue(headerValue string) string {
+	santizeHeaderValue := strings.ReplaceAll(headerValue, "'", `'\''`)
+	return "'" + santizeHeaderValue + "'"
```

Comment:
```suggestion
	sanitizeHeaderValue := strings.ReplaceAll(headerValue, "'", `'\''`)
	return "'" + sanitizeHeaderValue + "'"
```

Reply:
Hi @Miciah ! I dont get what is changed here.

Reply:
Missing `i` in san**i**tize.